README.md

```diff
@@ -147,6 +147,39 @@ request in the "Authorization" header.
       ...
     }
 
+### 5. Reseting the Access Token
```

Comment:
These numerical headers are meant for listing the required steps to use the library. The information you added is not required however, so I'd prefer that it was documented outside of this section. What about removing the numbering and moving these tips under a "Other features" header, etc.